README.md

```diff
@@ -236,7 +236,7 @@ and with a smooth evolutionary path.
 Safety, and especially
 [memory safety](https://en.wikipedia.org/wiki/Memory_safety), remain key
 challenges for C++ and something a successor language needs to address. Our
-initial priority and focus is on immediately addressing important, low-hanging
+initial priority and focus are on immediately addressing important, low-hanging
```

Comment:
```suggestion
initial priority and focus is on immediately addressing important, low-hanging
```

`is` seems more correct here "priority and focus" is referring to the same thing. Addressing ow-hanging fruit is our priority and focus.